packages/uniforms-bridge-json-schema/src/JSONSchemaBridge.ts

```diff
@@ -218,9 +219,7 @@ export default class JSONSchemaBridge extends Bridge {
     }
 
     if (type === 'array') {
-      const item = this.getInitialValue(joinName(name, '0'));
-      const items = props?.initialCount || 0;
-      return Array.from({ length: items }, () => item);
+      return [];
```

Comment:
We could actually use `minItems` and do a check, just like I did in https://github.com/vazco/uniforms/pull/1169/files#diff-fcbbc3f73ed7ff24460b9046c0e7b6dd430be61f1a18acbacabe75e5e0939239R93-R95. However, let's do it in a separate PR, to make easily reversible. (I'll raise this topic at our next meeting.)